test/controllers/AdminControllerSpec.scala

```diff
@@ -283,7 +306,7 @@ class AdminControllerSpec
       verify(mockSubmissionRepo, times(1)).save(expectedSubmission)
     }
 
-    "must must state to rejected when state is Submitted and Rejected request is sent" in {
+    "must state to rejected when state is Submitted and Rejected request is sent" in {
```

Comment:
"must **set** state to **Rejected** when state is Submitted and Rejected request is sent"

Reply:
done